### PR TITLE
[manageiq] modify plugin to check for modified files supplied

### PR DESCRIPTION
**Pull request by rh-dluong**
We have noticed an uptick in customers modifying appliance supplied code from our official repositories.  We would like to run an rpm -V against the following 5 packages to check for this to expedite the process of elimination for engineering and support:
        cfme
        cfme-appliance
        cfme-gemset
        cfme-appliance-tools
        cfme-appliance-common

[Bugzilla 1755080](https://bugzilla.redhat.com/show_bug.cgi?id=1755080)

Signed-off by: David Luong dluong@redhat.com

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [X] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [X] Is the subject and message clear and concise?
- [X] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [X] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

### CODE REVIEW

File: sos/plugins/manageiq.py [hunk 1]
```diff
@@ -84,4 +84,9 @@ def setup(self):
                     os.path.join(pg_dir, 'postgresql.conf')
             ])
 
+        self.collectExtOutput([
+            "rpm -V cfme cfme-appliance cfme-gemset
```
Comment (TurboTurtle) [inline sos/plugins/manageiq.py]: This is what the `--verify` option is for. Add the package names to the `packages` class member and when `--verify` is used, the package manager defined by the policy will run a verify command against all packages listed.

Comment (rh-dluong) [reply sos/plugins/manageiq.py]: Thanks for the feedback, made the change as requested.

**Comment by pmoravec** [comment]
Do you need to verify the packages on each and every run of sosreport (and `manageiq` plugin)? Then this change won't suffice. Either a sosreport `preset` can be created to automatically enable `--verify` option when a `cfme*` package is installed (seems legit), or the `rpm -V` call shall be added.

If you need to verify integrity of the packages on demand only, then ACK to the PR.

**Comment by rh-dluong** [comment]
@pmoravec Yeah, we'd like to run this everytime sos is ran.  In that case do I also need to modify redhat.py under policies?

**Comment by TurboTurtle** [comment]
@rh-dluong yes, presets get added in the policy for the distro(s) they should be available for.

Also, small nit: stylistically `packages` is a tuple, not list.

**Comment by pmoravec** [comment]
See https://github.com/sosreport/sos/blob/master/sos/policies/redhat.py#L197 and copy&replace relevant code with/around RH_SATELLITE.

> @pmoravec Yeah, we'd like to run this everytime sos is ran. In that case do I also need to modify redhat.py under policies?

Yes. See e.g. https://github.com/sosreport/sos/blob/master/sos/policies/redhat.py#L197 and copy&replace relevant code with/around all RH_SATELLITE references.

**Comment by rh-dluong** [comment]
Hello @pmoravec and @TurboTurtle , I have made the requested changes.

**Comment by rh-dluong** [comment]
Sorry about that, I tried to format it correctly but was guessing at it.  Should be fixed now, and the commit message has been updated.  Thanks @TurboTurtle

**Pull request closed without merging**